### DISCUSSION TITLE
Multiple Instances

### DISCUSSION THREAD

**Discussion by virgo77** [azerothcore/azerothcore-wotlk#11317] [General]
Hi All!
I just have a little question for you, before trying this.

I know that it is possible to run multiple instances of the core, on condition that each core uses separated databases.

What I would like to try is this configuration:
- 1 AzerothCore without any mods (Debian)
- 1 AzerothCore with mods (Windows)
- 4 Tables: auth / characters (used by both cores), world (for the 1st core) and world.modded (for the 2nd core)

So each core can use the same Auth service and DB, and can access the same characters table.
The goal is that I can create accounts through the [WoWSimpleRegistration](https://github.com/masterking32/WoWSimpleRegistration) page and the accounts can be used even I play on the Modded Core or not.

Is it possible and working? Or can each core crash because an another one is connected at the same time on the Auth/Chars tables?

I just need advices :)
Yep I know, I have some strange questions sometimes ^^

Thanks a lot for your answers

V.

**Comment by Nyeriah** [comment]
Yes, you can't have two servers connected to the same character database, because certain operations lock the tables and that will inevitably lead to issues, same as concurrent queries.

For auth it's fine, it's designed to support multiple realms

**Reply by virgo77** [reply]
erf, I was sure you would say that ^^
Ok, I understand.
So thanks a lot for your reply

Have a nice day :)

V.
